Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853897): Gdk-CRITICAL **: 15:34:57.157: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852780): Gdk-CRITICAL **: 15:34:57.608: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853131): Gdk-CRITICAL **: 15:34:58.854: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853208): Gdk-CRITICAL **: 15:34:58.924: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853681): Gdk-CRITICAL **: 15:34:59.193: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854202): Gdk-CRITICAL **: 15:34:59.237: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854337): Gdk-CRITICAL **: 15:34:59.245: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852812): Gdk-CRITICAL **: 15:34:59.266: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852694): Gdk-CRITICAL **: 15:34:59.270: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852914): Gdk-CRITICAL **: 15:34:59.297: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853195): Gdk-CRITICAL **: 15:34:59.306: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853928): Gdk-CRITICAL **: 15:34:59.329: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853284): Gdk-CRITICAL **: 15:34:59.385: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853649): Gdk-CRITICAL **: 15:34:59.407: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853892): Gdk-CRITICAL **: 15:34:59.426: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853395): Gdk-CRITICAL **: 15:34:59.465: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852944): Gdk-CRITICAL **: 15:34:59.943: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854233): Gdk-CRITICAL **: 15:34:59.971: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852975): Gdk-CRITICAL **: 15:35:00.022: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854055): Gdk-CRITICAL **: 15:35:00.051: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853299): Gdk-CRITICAL **: 15:35:00.059: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854454): Gdk-CRITICAL **: 15:35:00.115: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852857): Gdk-CRITICAL **: 15:35:00.143: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854371): Gdk-CRITICAL **: 15:35:00.550: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853622): Gdk-CRITICAL **: 15:35:01.569: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853015): Gdk-CRITICAL **: 15:35:01.582: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854150): Gdk-CRITICAL **: 15:35:01.702: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853565): Gdk-CRITICAL **: 15:35:01.757: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852583): Gdk-CRITICAL **: 15:35:02.418: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854578): Gdk-CRITICAL **: 15:35:03.703: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853799): Gdk-CRITICAL **: 15:35:03.814: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1853706): Gdk-CRITICAL **: 15:35:03.949: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852902): Gdk-CRITICAL **: 15:35:04.197: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853236): Gdk-CRITICAL **: 15:35:04.205: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853037): Gdk-CRITICAL **: 15:35:04.243: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853414): Gdk-CRITICAL **: 15:35:04.273: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853779): Gdk-CRITICAL **: 15:35:04.297: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854537): Gdk-CRITICAL **: 15:35:04.316: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854548): Gdk-CRITICAL **: 15:35:04.506: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853079): Gdk-CRITICAL **: 15:35:04.642: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852821): Gdk-CRITICAL **: 15:35:04.716: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1853448): Gdk-CRITICAL **: 15:35:04.779: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853151): Gdk-CRITICAL **: 15:35:05.076: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852752): Gdk-CRITICAL **: 15:35:05.799: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854571): Gdk-CRITICAL **: 15:35:06.282: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854493): Gdk-CRITICAL **: 15:35:07.022: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853165): Gdk-CRITICAL **: 15:35:07.541: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852704): Gdk-CRITICAL **: 15:35:08.483: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853844): Gdk-CRITICAL **: 15:35:08.578: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854295): Gdk-CRITICAL **: 15:35:08.922: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853378): Gdk-CRITICAL **: 15:35:08.973: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854589): Gdk-CRITICAL **: 15:35:09.054: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853667): Gdk-CRITICAL **: 15:35:09.084: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853999): Gdk-CRITICAL **: 15:35:09.333: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854585): Gdk-CRITICAL **: 15:35:09.826: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853943): Gdk-CRITICAL **: 15:35:09.893: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1854566): Gdk-CRITICAL **: 15:35:09.907: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853039): Gdk-CRITICAL **: 15:35:10.005: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853995): Gdk-CRITICAL **: 15:35:10.085: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853759): Gdk-CRITICAL **: 15:35:10.135: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852556): Gdk-CRITICAL **: 15:35:10.164: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854067): Gdk-CRITICAL **: 15:35:10.173: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed

(DataCollector_Deterministic.py:1853679): Gdk-CRITICAL **: 15:35:10.184: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853004): Gdk-CRITICAL **: 15:35:10.420: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854229): Gdk-CRITICAL **: 15:35:10.544: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854577): Gdk-CRITICAL **: 15:35:10.800: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853312): Gdk-CRITICAL **: 15:35:11.331: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852886): Gdk-CRITICAL **: 15:35:11.455: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852979): Gdk-CRITICAL **: 15:35:11.660: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854393): Gdk-CRITICAL **: 15:35:11.690: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853911): Gdk-CRITICAL **: 15:35:11.757: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854480): Gdk-CRITICAL **: 15:35:12.194: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853653): Gdk-CRITICAL **: 15:35:12.398: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853101): Gdk-CRITICAL **: 15:35:12.426: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853420): Gdk-CRITICAL **: 15:35:12.618: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852779): Gdk-CRITICAL **: 15:35:12.620: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854277): Gdk-CRITICAL **: 15:35:12.640: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852537): Gdk-CRITICAL **: 15:35:12.644: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853250): Gdk-CRITICAL **: 15:35:12.648: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854107): Gdk-CRITICAL **: 15:35:12.663: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852651): Gdk-CRITICAL **: 15:35:12.672: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852745): Gdk-CRITICAL **: 15:35:12.708: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853391): Gdk-CRITICAL **: 15:35:12.721: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852674): Gdk-CRITICAL **: 15:35:12.724: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853901): Gdk-CRITICAL **: 15:35:12.732: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853243): Gdk-CRITICAL **: 15:35:12.740: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854177): Gdk-CRITICAL **: 15:35:12.744: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854568): Gdk-CRITICAL **: 15:35:12.759: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852606): Gdk-CRITICAL **: 15:35:12.784: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853873): Gdk-CRITICAL **: 15:35:12.812: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854556): Gdk-CRITICAL **: 15:35:12.824: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854357): Gdk-CRITICAL **: 15:35:12.840: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853561): Gdk-CRITICAL **: 15:35:12.848: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854123): Gdk-CRITICAL **: 15:35:12.868: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854397): Gdk-CRITICAL **: 15:35:12.916: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853529): Gdk-CRITICAL **: 15:35:12.920: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854563): Gdk-CRITICAL **: 15:35:12.928: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1852631): Gdk-CRITICAL **: 15:35:12.976: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1854119): Gdk-CRITICAL **: 15:35:13.031: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Unable to init server: Could not connect: Connection refused

(DataCollector_Deterministic.py:1853442): Gdk-CRITICAL **: 15:35:13.068: gdk_cursor_new_for_display: assertion 'GDK_IS_DISPLAY (display)' failed
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...

Data Dump...
Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.Initalising...
Running Sim...
.....................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
[1727 8682]
....
[1739 8670]
...
[1996 8403]
.....
[1712 8686]
.
[1679 8670]
.
[1750 8630]
.
[1693 8681]
..
[1701 8698]
..
[1696 8654]
..
[2368 8021]
.
[2031 8339]
.
[2407 7979]
.
[1671 8722]
..
[1998 8393]
.
[1724 8649]
.
[1785 8590]
.
[1800 8567]
.
[1775 8598]
.
[1726 8669]
..
[1756 8664]
.
[1714 8651]
.
[1728 8641]
.
[1775 8615]
.
[1721 8674]
.
[1710 8642]
.
[1778 8579]
.
[1769 8599]
.
[1731 8627]
.
[1892 8503]
.
[1803 8584]
.
[1775 8619]
.
[1724 8657]
..
[1732 8650]
.
[1939 8457]
..
[1786 8615]
.
[1765 8645]
.
[1808 8566]
..
[1890 8491]
..
[1729 8683]
.
[1750 8628]
.
[1766 8638]
..
[1904 8500]
.
[1989 8403]
.
[1946 8445]
.
[1951 8405]
..
[1705 8652]
.
[1718 8633]
.
[1754 8636]
.
[1839 8577]
.
[1955 8486]
.
[2296 8090]
..
[1713 8652]
.
[1734 8631]
.
[2273 8075]
.
[1737 8647]
.
[1726 8687]
.
[1800 8619]
..
[2397 8029]
..
[2184 8205]
....
[2407 7952]
..
[2674 7713]
.
[2264 8107]
.
[1922 8453]
..
[2516 7900]
....
[2323 8015]
.
[2459 7887]
.
[2474 7903]
.
[2542 7820]
.
[2376 8034]
..
[2944 7429]
..
[2832 7559]
.
[3095 7245]
.
[2391 7982]
..
[3649 6755]
.
[2744 7650]
.
[2214 8192]
.
[2723 7665]
...........
[3492 6845]
.....
[3048 7280]
..........
[3456 6917]
............................
[3863 6542]
...
[3468 6928]
.......
[6860 3473]
.
[3452 6918]
..
[4617 5769]
..
[5266 5056]
...
[4728 5620]
......................
[5867 4477]
.....................
[5737 4563]
....
[5529 4829]
..................................................................................................................................................................................................................................................................
Data Dump...
........
Data Dump...
.....
Data Dump...
..
Data Dump...
.......
Data Dump...

Data Dump...

Data Dump...
.............
Data Dump...
.................
Data Dump...
.............................................................................................
[  47 2401]
....................................
[  60 2456]
..
[  44 2395]
.....................
[  61 2442]
.
Data Dump...
...........................
[  54 2601]
...
Data Dump...
.
[  49 2607]
...................
[  57 2423]
............
[  72 2556]
.
Data Dump...

[  57 2581]
........
[  54 2650]
.......
[ 127 2313]
............
[  63 2512]
.......
[  76 2445]
....
Data Dump...
.
[  65 2477]
....
[  81 2547]
.........
[  60 2641]
.
[  84 2448]
....
[  63 2601]
..
[ 103 2511]
.
[  74 2345]
.........
[  81 2554]
...
[  80 2390]
..
[  98 2399]
...............
Data Dump...
..
[  96 2452]
.
[ 253 1985]
.
[ 108 2308]
.....
[ 110 2303]
...
[ 103 2661]
.
[ 101 2343]
...
[ 112 2624]
..
[  85 2550]
..............
[ 145 2420]
.........
[ 139 2331]
...
[ 134 2284]
......
[ 134 2511]
.
[ 122 2511]
.....
[  98 2474]
.........
Data Dump...
..
Data Dump...
.
[ 357 1807]
.........................
[ 163 2131]
..
Data Dump...
..
[ 528 1606]
.........
[ 272 2008]
...
Data Dump...
................
Data Dump...
.......
[ 707 1149]
....
[ 214 2483]
....
Data Dump...

[ 250 2038]
...................
Data Dump...
.
[ 222 2198]
.
Data Dump...

[ 235 2303]
.................
Data Dump...
............
[ 293 1809]
.........
[ 255 2289]
.......
[ 395 1622]
...
[ 394 1445]
....
[ 365 1882]
....
[ 337 2036]
.
[ 441 1894]
..
Data Dump...
......
Data Dump...
.....
Data Dump...
...
Data Dump...
....................
[ 494 1582]
.
Data Dump...

[ 333 2005]
..
Data Dump...
...
[ 464 1730]
......
[ 446 1955]
..
[ 610 1246]
...
Data Dump...
.
[1029 1134]
......
Data Dump...
.....
[ 552 1659]
...
[ 557 1412]
....
[ 645 1588]
....
Data Dump...
.
[ 613 1350]
.........
[ 654 1439]
.
[ 809 1019]
.
Data Dump...
........
Data Dump...

Data Dump...
..
[ 769 1085]
....
[2167  170]
..
[ 559 1452]
..
Data Dump...
.......
Data Dump...
.......
[1003  869]
..
Data Dump...

[1193  714]
.....
[ 880 1107]
...
Data Dump...
.......
Data Dump...
....
[1137  807]
.........
[1054 1074]
....
Data Dump...

Data Dump...

Data Dump...
..
[ 793 1508]
...
Data Dump...

Data Dump...

[1866  356]
..
[1741  317]
...........
[1306  929]
.
[1423  776]
.
[1903  229]
..................
[ 981 1020]
......
[1786  404]
.....
Data Dump...
...............................
Data Dump...
..........
Data Dump...
..........................
Data Dump...
.
Data Dump...
..............
Data Dump...

Data Dump...
..........
Data Dump...
........
Data Dump...
.....................
Data Dump...
................
Data Dump...
...........
[27 16]

Data Dump...
..
Data Dump...
..
Data Dump...
.............
Data Dump...
.........
Data Dump...
..
Data Dump...
....
Data Dump...
...............
Data Dump...
....
Data Dump...
...
Data Dump...
............
[34 16]

Data Dump...
.....
Data Dump...
.
[26 38]

Data Dump...
......
[48 40]

Data Dump...
.....
[34 27]

Data Dump...
........
[51 24]

Data Dump...
..
[35 27]

Data Dump...
.
[51 23]

Data Dump...
......
[28 34]

Data Dump...
....
[82 12]

Data Dump...
.
[41 18]

Data Dump...
.......
[33 15]

Data Dump...

[53 22]

Data Dump...
...
[50 18]

Data Dump...
......
[56 28]

Data Dump...
.
[44 40]

Data Dump...

[46 14]

Data Dump...
.......
[88 15]

Data Dump...

[61 31]

Data Dump...

[84 14]

Data Dump...

[78 19]

Data Dump...

[78 28]

Data Dump...
..
[80 17]

Data Dump...

[40 26]

Data Dump...
High 1 7
Done
